sap-extensions/v2.0-extensions.json

```diff
@@ -32,6 +32,49 @@
           "type": "boolean",
           "default": false,
           "description": "Indicates if the marked object represents a root entity. A root entity is the root of a business object. It has a distinct existence. Examples are real-world objects, like a workforce person or a piece of equipment or other more abstract real-world concepts, like a contract or an order. It has an independent lifetime / identity and is globally and uniquely addressable (via the unique identifier provided by the property referenced in x-sap-odm-oid field)."
+        },
+        "x-sap-dpp-entitySemantics": {
```

Comment:
We have a convention for extension fields names that they must follow kebab-case-notation.

There are some old fields with camelCase notation, those are exceptions due to historical reasons.